src/collective/cover/tests/test_views.py

```diff
@@ -81,10 +82,12 @@ def test_body_class(self):
 
     # @@updatetile
     def test_update_tile_view(self):
+        # This view should be available to Anonymous users
+        logout()
         # valid tile-id parameter returns tile
         tile_id = self.c1.list_tiles()[0]  # use a tile on the layout
         self.request.form['tile-id'] = tile_id
-        view = api.content.get_view(u'updatetile', self.c1, self.request)
+        view = self.c1.restrictedTraverse('@@updatetile')
```

Comment:
I need to ensure this view is available to Anonymous users.  ``api.content.get_view`` keep working if I change the view permission to ``cmf.ModifyPortalContent``